client/flags/flags.go

```diff
@@ -62,6 +62,7 @@ const (
 	FlagPageKey          = "page-key"
 	FlagOffset           = "offset"
 	FlagCountTotal       = "count-total"
+	FlagTimeoutHeight    = "timeout-height"
```

Comment:
I Agree that would nice, but I'll leave it out of this PR for now. The way to do this would be to have a pre-run cmd handler that intercepts this flag and attempts a query, returning an error upon failure.